docs/how-it-works.md

```diff
@@ -0,0 +1,138 @@
+---
+sidebar_position: 2
+---
+
+# How It Works
+
+Learn how Divvi Up works and how it protects privacy.
+
+## Breakdown
+
+Divvi Up ingests measurements and aggregates them, while protecting the privacy
+of each individual measurement. This can be broadly broken down into several
+steps, described in the following sections.
+
+<!-- TODO(inahga): Consider making this a more detailed `mermaid` diagram -->
+
+![](../static/how-it-works.png#gh-dark-mode-only)
+![](../static/how-it-works-light.jpg#gh-light-mode-only)
+
+See the [Distributed Aggregation Protocol (DAP)][DAP] for a highly detailed
+specification of the protocol that underpins Divvi Up.
+
+### Step 0: Subscriber sets up Divvi Up
+
+Subscribers are users of Divvi Up, such as app developers, who wish to collect
+quantitative measurements about their end-user applications.
+
+The subscriber chooses a DAP client library, which is used to interact with any
+DAP-compatible measurements aggregator, such as the aggregators provided by
+Divvi Up and its partners. Client libraries for [Rust
+(janus_client)][janus_client] and [TypeScript (divviup-ts)][divviup-ts] are
+available.
+
+For each measurement the subscriber wishes to aggregate, the subscriber creates
+a task. A task encodes metadata about how the measurement will be made, which
```

Comment:
nit: I wouldn't say "encode", because that implies some specific wire representation of a message, and that's not the sense of a task we're discussing here. Maybe "A task contains" or "A task consists of".